### ISSUE TITLE
64gb for semantic3d

### ISSUE THREAD

**Issue by a18700** [QingyongHu/RandLA-Net#143]
Hello, thanks for great work first.

One of best works i ever saw, but i have minor concerns on replicating your results.

1. To replicate your semantickitti results, the train step should be set to (len(numb point cloud) over batchsize) or 200(manually set to small number) can be employed?

2. I really want to implement your model with semantic3d dataset, but i've found out that gpus with large memory are required to run it with single gpu.

In my environment, unfortunately, we have multiple gpus with limited memory, so i wonder if there is aimplementation for 'synchronized batch norm-keeped version of yours.'

I appreciate your endeavor for answering the question.
Thanks.
